### ISSUE TITLE
Rule: link-name changes

### ISSUE THREAD

**Issue by jkodu** [dequelabs/axe-core#1147]
The follow test case should fail.

Scenario: Not exposed to assistive technologies.
```html
<a href="http://www.w3.org/WAI" aria-hidden="true"><img src="#" /></a>
```

**Comment by WilcoFiers** [comment]
I think this should fail unless it's not focusable.

**Comment by jkodu** [comment]
@WilcoFiers 
Was it this one that was going to be a discussion for Auto WCAG?

**Comment by WilcoFiers** [comment]
I think it should be. IMO, this should fail. Please raise an issue in Auto-WCAG instead.

**Comment by dylanb** [comment]
We have been discussing for some time adding a rule that would fail tab focusable things inside aria-hidden="true" regions. Even if this link had an accessible name, it will cause screen reader problems becuase of the aria-hidden="true"
